onnxruntime/python/tools/transformers/fusion_embedlayer.py

```diff
@@ -179,6 +181,7 @@ def fuse(self, node, input_name_to_nodes, output_name_to_node):
             if segment_path is None:
                 return
             else:
+                assert onnxruntime.__version__ > '1.4.0', "Please install onnxruntime with version > 1.4.0 for distilbert fusion support"
```

Comment:
return when onnxruntime version is 1.4.0 or older; Use packaging.version to parse version.